pkgs/applications/video/vlc/default.nix

```
@@ -52,6 +53,7 @@ stdenv.mkDerivation rec {
     fluidsynth wayland wayland-protocols
   ] ++ optional (!stdenv.hostPlatform.isAarch64) live555
     ++ optionals withQt5    [ qtbase qtsvg qtx11extras ]
+    ++ optionals skins2Support (with pkgs; [ xorg.libXpm freetype xorg.libXext xorg.libXinerama ])
```

```suggestion
    ++ optionals skins2Support [ libXpm freetype libXext libXinerama ])
```

Something like that. Please test this.